### ISSUE TITLE
Where can i get the pretrained model?or should I train from scratch?

### ISSUE THREAD

**Issue by xtyawesome** [bahetibhakti/Distracted-Driver-Detection#1]
Thanks for contributing a project like this.
I want to improve the accuracy using this paper's neural network start with retrain this project.
But i cannot get the pretrained model .
Can you please share weights.h5 with me. 
It will really helps a lot.
Thanks again.
